code/modules/surgery/organs/liver.dm

```diff
@@ -0,0 +1,15 @@
+/obj/item/organ/liver
+	name = "liver"
+	icon_state = "liver"
+	origin_tech = "biotech=5"
+	w_class = WEIGHT_CLASS_NORMAL
+	zone = "chest"
+	slot = "liver"
+	desc = "Goes great with a nice chianti and some fava beans."

```

Comment:
Perhaps you should add some iron reagent to this if eaten, because "Liver is high in iron" meme.